proposals/0152-precompiles.md

```diff
@@ -0,0 +1,273 @@
+---
+simd: '0152'
+title: Precompiles
+authors:
+  - Emanuele Cesena
+category: Standard
+type: Core
+status: Draft
+created: 2024-06-03
+feature:
+supersedes:
+superseded-by:
+extends:
+---
+
+## Summary
+
+Define a unified behavior for precompiles, and highlight a few minor
+changes to the existing precompiles to minimize differences.
+
+## Motivation
+
+Precompiles are special native programs designed to verify additional
+transaction signatures.
+They run without the VM and without loading any account, and they can access
+data from other instructions, within the same transaction.
```

Comment:
If I remember correctly, signatures verified from a precompile are also charged in the base fee and don't require compute units. Maybe a statement like that doesn't belong in this document.

Reply:
We have the same sentence in the specs below. I added a comment about cost there, instead of in the motivation.